Wrapping/Generators/Python/itk/__init__.py

```diff
@@ -70,8 +70,10 @@ def _get_lazy_attributes(local_lazy_attributes, l_module, l_data):
             lzy_module = _itkLazy.LazyITKModule(__name__, lazy_attributes)
             # Set the __path__ attribute, which is required for this_module
             # to be used as a package
-            setattr(lzy_module, "__path__", __path__)
-            setattr(lzy_module, "__spec__", __spec__)  # pytype: disable=name-error
+            setattr(lzy_module, "__path__", sys.modules[__name__].__path__)
```

Comment:
@thewtex  NOTE:  These are still set, but now preferring the fully qualified name to avoid a pytype failure when using the implicit __spec__